From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 13 Sep 2026 15:10:00 +0200
Subject: [PATCH] sysbus: grant the HFP AG the telephony query ACG

With outbound to telephony permitted, the AG still needs a group that
covers the two queries webos-telephonyd actually implements:

  src/AG/hfpagsubscribe.h:97  com.palm.telephony/signalStrengthQuery
  src/AG/hfpagsubscribe.h:98  com.palm.telephony/networkStatusQuery

Both sit in telephony.query and telephony.management; telephony.query is
the read-only one and is all the gateway needs.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending
---
 files/sysbus/com.webos.service.hfp.perm.json   | 3 ++-
 1 file changed, 2 insertion(s), 1 deletion(s)

--- a/files/sysbus/com.webos.service.hfp.perm.json
+++ b/files/sysbus/com.webos.service.hfp.perm.json
@@ -1,3 +1,4 @@
 {
-    "com.webos.service.hfp": ["bluetooth.query","bluetooth.management","bluetooth.operation","account.query","audio.operation"]
+    "com.webos.service.hfp": ["bluetooth.query","bluetooth.management","bluetooth.operation","account.query","audio.operation",
+"telephony.query"]
 }
